pkg/webui/oauth/views/login/index.js

```diff
@@ -60,7 +60,6 @@ const appRoot = selectApplicationRootPath()
 @bind
 export default class OAuth extends React.PureComponent {
   static propTypes = {
-    location: PropTypes.string.isRequired,
```

Comment:
It is used in the component: https://github.com/TheThingsNetwork/lorawan-stack/blob/8cd545e3df2299019599b4fd52ca62f7c04c414d/pkg/webui/oauth/views/login/index.js#L77

Reply:
ah yeah, will update